functions-and-scope.js

```diff
@@ -42,6 +65,15 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 
 // ---- Verwachte uitkomst: 6.642857142857143
 
+let addedGrades = 0;
+
+for (let i = 0; i < grades.length; i++) {
+    addedGrades = grades[i] + addedGrades;
```

Comment:
Je kan dit versimpelen:
```suggestion
    addedGrades += grades[i];
```